.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "web/slide_puzzle"]
+	path = web/slide_puzzle
+	url = git@github.com:kevmoo/slide_puzzle.git
```

Comment:
you'll have to use an https URI to make travis happy – or anyone else who is not authenticated

Reply:
ah, good catch